galaxygpt-api/Program.cs

```diff
@@ -199,4 +201,10 @@ public class AskResponse
     public required string Context { get; init; }
     public required string Duration { get; init; }
     public required string Version { get; init; }
+    
+    [JsonPropertyName("question_tokens")]
+    public required string QuestionTokens { get; init; }
+    
+    [JsonPropertyName("response_tokens")]
+    public required string ResponseTokens { get; init; }
```

Comment:
add some xmldoc explaining these. it's important to do so since it'll show up on the swagger api docs which get auto generated.